src/Models/PlayerCardModel.php

```diff
@@ -101,7 +107,18 @@ public function addNewCard(PlayerCard $newPlayer):bool {
             ':Position' => $newPlayer->PositionName,
             ':Control' => $newPlayer->Control,
             ':Attack' => $newPlayer->Attack,
-            ':Defence'=> $newPlayer->Defence
+            ':Defence' => $newPlayer->Defence
         ]);
     }
+
+    public function changeActivationStatus(int $id, int $Deleted)
+    {
+        $query = $this->db->prepare("UPDATE `PremierLeagueCards`
+        SET `Deleted` = $Deleted
```

Comment:
You should use parameter binding here to be safe, the data doesn't currently come from a user, but for the sake of a minute or two of extra work it's better to be cautious